agent/engine/docker_container_engine.go

```diff
@@ -271,7 +274,7 @@ func (dg *dockerGoClient) pullImage(image string, authData *api.RegistryAuthenti
 
 	// Special case; this image is not one that should be pulled, but rather
 	// should be created locally if necessary
-	if image == emptyvolume.Image+":"+emptyvolume.Tag {
+	if image == fmt.Sprintf(imageNameFormat, emptyvolume.Image, emptyvolume.Tag) {
```

Comment:
Why was this changed?

Reply:
I thought this was better the previous way of doing this, and we will use this format for more internal image names.